rest/resource-server/src/docs/asciidoc/licenses.adoc

```diff
@@ -48,3 +48,25 @@ include::{snippets}/should_document_get_license/http-response.adoc[]
 
 ===== Links
 include::{snippets}/should_document_get_license/links.adoc[]
+
+[[resources-license-link-obligation]]
+==== Link obligations to license
+
+A `POST` request will link obligations to a license.
```

Comment:
It would be good to mention explicitly to pass list of obligations as there is no request structure.

Reply:
Explanation for parameters was added follow your comment.